ansible/install/grafana-dashboards.yml

```diff
@@ -3,7 +3,8 @@
 # Playbook to upload CFME/Miq Grafana Performance Dashboards to Grafana host
 #
 
-- hosts: localhost
+# - hosts: localhost
+- hosts: monitorhost
```

Comment:
Yes I'm still working on this part. Somehow something wasn't working and I need to go back and check what it was.